### PR TITLE
Add Data.ByteString

### PR DESCRIPTION
**Pull request by saralich**
Story details: https://app.clubhouse.io/itprotv/story/52930

### CODE REVIEW

File: Prolude/ByteString.hs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+where
+
+import Data.ByteString (ByteString, writeFile)
+import qualified Data.ByteString.Lazy as LBS
+import qualified System.IO as IO
+
+type LazyByteString = LBS.ByteString
+
+putLazyStr :: LazyByteString -> IO.IO ()
+putLazyStr = LBS.putStr
```
Comment (tfausak) [inline Prolude/ByteString.hs:15]: Is this used enough to warrant inclusion? 

Also: Do we want to `liftIO` it?

Comment (saralich) [reply Prolude/ByteString.hs:15]: There are 20 uses of it, though all in Scripts, and I'm in favor of liftIO-ing it!

Comment (tfausak) [reply Prolude/ByteString.hs:15]: Also also: What do you think about calling it `putLazyByteString`?

File: Prolude/ByteString.hs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+module Prolude.ByteString
+    ( module Data.ByteString
+    , LazyByteString
+    , putLazyStr
+    )
+where
+
+import Data.ByteString (ByteString, writeFile)
```
Comment (tfausak) [inline Prolude/ByteString.hs:8]: We should rename `writeFile` to something else, like `writeLazyByteStringToFile` (or maybe something less clunky).

**Review by tfausak — APPROVED** [review]

**Pull request merged**